tests/bluetooth/controller/ctrl_ull_dletime/src/dle_2m.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2018-2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define FEATURES_TO_TEST 4
+#define OCTETS_TO_TEST 5
+
+static u16_t default_octets[] = {27, 60, 128, 200, 251};
+
+static u16_t feature[] = {0,
+		BIT(BT_LE_FEAT_BIT_PHY_2M),
+		BIT(BT_LE_FEAT_BIT_PHY_CODED),
+		BIT(BT_LE_FEAT_BIT_PHY_2M) || BIT(BT_LE_FEAT_BIT_PHY_CODED)};
```

Comment:
This looks like a bug: shouldn't it be `|` instead of `||`?

Reply:
yes